src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java

```diff
@@ -539,7 +538,32 @@ public void visualizzazioneSezioneAltriRecapiti() {
     public void visualizzazioneSezioneAltriRecapitiPG() {
         vaiInFondoAllaPagina();
         By altriRecapitiSectionBy = By.id("specialContactTitle");
-        getWebDriverWait(5).withMessage(" Non si visualizza correttamente  il titolo della sezione altri recapiti").until(ExpectedConditions.visibilityOfElementLocated(altriRecapitiSectionBy));
+        By titleGiaAssociatiBy = By.xpath("//p[contains(text(), 'Già associati')]");
+        WebElement tableGiaAssociati = driver.findElement(By.xpath("//table[@aria-label='Già associati']"));
+        By modifyButtonBy = By.xpath(".//button[contains(@id, 'modifyContact')]");
+        By deleteButtonBy = By.xpath(".//button[contains(@id, 'cancelContact')]");
```

Comment:
If there are multiple element with the same id, you can just do a specific select like so: 
`By.xpath("//div[@id='example']//div[@id='sub-example']//button[@id='real-element']")`